pyproject.toml

```diff
@@ -15,6 +15,7 @@ pexpect = "^4.8.0"
 ipykernel = "^5.3.0"
 jupyter_client = "^6.1.3"
 ipython = "^7.15.0"
+regex = "^2021.4.4"
```

Comment:
What syntax the builtin re module does not have compared to regex?
I want to know why you add this dependencies.

Reply:
Finally, I got. you want to use `(?R)` right?

Reply:
Yes.